### PR TITLE
Create arize_llm_observability.ipynb

### PR DESCRIPTION
**Pull request by hakantekgul**
Adding documentation and notebook for Arize callback handler. 

  - @dev2049
  - Agents / Tools / Toolkits: @vowelparrot
  - Tracing / Callbacks: @agola11

### CODE REVIEW

File: docs/extras/ecosystem/integrations/arize_llm_observability.ipynb [hunk 1]
```diff
@@ -0,0 +1,1765 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Awn1jcNa0lLZ"
+      },
+      "source": [
+        "# Arize <> LangChain Integration\n",
```
Comment (hwchase17) [inline docs/extras/ecosystem/integrations/arize_llm_observability.ipynb:9]: for consistency with other integrations, lets just call the title `Arize`

**Review by hwchase17 — COMMENTED** [review]
one nit, otherwise lgtm!

File: docs/extras/ecosystem/integrations/arize_llm_observability.ipynb (hunk 1 above)
Comment (hakantekgul) [reply docs/extras/ecosystem/integrations/arize_llm_observability.ipynb]: Done!

**Comment by hakantekgul** [comment]
Ready for merge! @hwchase17

**Pull request merged**